js/components/tabs.js

```diff
@@ -27,6 +27,25 @@ class Tabs extends ImmutableComponent {
     this.onNextPage = this.onNextPage.bind(this)
     this.onNewTabLongPress = this.onNewTabLongPress.bind(this)
     this.wasNewTabClicked = this.wasNewTabClicked.bind(this)
+    this.onMouseLeave = this.onMouseLeave.bind(this)
+    this.onFrameClose = this.onFrameClose.bind(this)
+
+    this.state = {
+      fixTabWidth: null
+    }
+  }
+
+  onMouseLeave () {
+    this.setState({
+      fixTabWidth: null
+    })
+    this.forceUpdate()
```

Comment:
We should avoid using `forceUpdate()` to force re-render, it's ill advised by React itself. 

Ref: https://facebook.github.io/react/docs/react-component.html#forceupdate

> Normally you should try to avoid all uses of forceUpdate() and only read from this.props and this.state in render().

Could you do it in a way to avoid that? thanks!